Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.xaml

```diff
@@ -1,38 +1,38 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:contract5NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,5)"
-                    xmlns:contract5Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,5)"
                     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
-                    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+                    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:contract5Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,5)"
+    xmlns:contract5NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,5)">
 
     <ResourceDictionary.ThemeDictionaries>
+        <!-- Default is a fallback if a more precise theme isn't called
+            out below -->
         <ResourceDictionary x:Key="Default">
-            <contract5Present:AcrylicBrush x:Key="MenuFlyoutAcrylicBackgroundBrush"
-                                           BackgroundSource="Backdrop"
-                                           FallbackColor="{ThemeResource SystemChromeMediumLowColor}"
-                                           TintColor="{ThemeResource SystemChromeMediumLowColor}"
-                                           TintOpacity="0.8" />
-        </ResourceDictionary>
+            <SolidColorBrush x:Key="SystemControlButtonBackgroundPointerOver" Color="{ThemeResource SystemChromeHighColor}" />
```

Comment:
We should use StaticResource instead of ThemeResource in both Light and Dark. It's OK to use ThemeResource in HighContrast.